code/modules/cargo/packs.dm

```diff
@@ -364,12 +364,12 @@
 	crate_name = "stun baton crate"
 
 /datum/supply_pack/security/taser
-	name = "Taser Crate"
-	desc = "From the depths of stunbased combat, this order rises above, supreme. Contains three hybrid tasers, capable of firing both electrodes and disabling shots. Requires Security access to open."
+	name = "Disabler Crate"
+	desc = "From the depths of less stunbased combat, this order rises above, supreme. Contains three hybrid tasers, capable of firing both electrodes and disabling shots. Requires Security access to open."
```

Comment:
>Contains three hybrid tasers

🤔

Reply:
a disabler crate already exists actually looking more than just find and replace, I'm just deleting this one.

Reply:
keep the taser crate but make the price 20K to encourage big capatilism

Reply:
Honestly that sounds like it'd just make guncargo overpowered compared to the rest of the station, not sure it's a good idea. Especially since this means miners, already a powerful department, would have easier access to tasers than others.

Reply:
"capable of firing both electrodes and disabling shots" Might want to fix that.

Reply:
this crate already got destroyed